tgui/packages/tgui/interfaces/Vending.tsx

```diff
@@ -226,84 +241,84 @@ const ProductImage = (props) => {
       src={`data:image/jpeg;base64,${product.img}`}
       style={{
         'vertical-align': 'middle',
+        /** Horizontal align on typescript doesnt work on our codebase. Great. */
```

Comment:
Still cannot get horizontal alignment to work in typescript files. Something's messed up with tgui.

If you have an idea on how to fix, let me know, because its bad in the long run. Right now however, I think the menu looks fine

Reply:
Horizontal align doesnt exist. To center just text you can use `text-align: center;`. To center images and other content, you can use:
```
margin-left: auto;
margin-right: auto;
```

Reply:
Gotcha. Thanks